vispy/visuals/text/text.py

```diff
@@ -206,8 +206,10 @@ class TextVisual(Visual):
 
     Parameters
     ----------
-    text : str
-        Text to display.
+    text : str | list of str
+        Text to display. Can also be a list of strings.
+        Note: support for list of str will likely be removed soon
+        in favor of text collections.
```

Comment:
Yeah that's true. I don't want to lock us into an API until we've seen how all the collections stuff will work, though. I'll change the comment here to say that it might be removed in favor of collections, instead of probably will.